styles/colors.js

```diff
@@ -13,6 +13,15 @@ export const colors = {
 		dark: 'rgba(46, 46, 46, 1)',
 		light: 'rgba(255, 255, 255, 1)',
 	},
+	gray: {
```

Comment:
I don't think this is good organization. This file should be organized by asset, not by color.


For example, instead of writing...

```
gray: {
    text: [value]
    // other assets
}
```

I think we should go with...

```
text: {
    dark: [value],
    light: [value],
    gray: [value]
}
```